src/components/Sidebar.js

```diff
@@ -52,33 +57,35 @@ class Sidebar extends Component {
                 <img alt="avatar" src={user.avatarURL} />
               </div>
               <div className="info">
-                <a data-toggle="collapse" href="#collapseExample" class="collapsed">
+                <a data-toggle="collapse" href="#collapseExample" className="collapsed">
                   <span>
                     {renderedUserName}
                     <b className="caret"></b>
                   </span>
                 </a>
                 <div className="clearfix"></div>
                 <div className="collapse" id="collapseExample">
-                  <ul class="nav">
-                      <li>
-                          <Link to={`/profile/${user.username}`} exact>
-                              <span class="sidebar-mini"> MP </span>
-                              <span class="sidebar-normal"> My Profile </span>
-                          </Link>
-                      </li>
-                      <li>
-                          <Link to={`/profile/${user.username}`} exact>
-                              <span class="sidebar-mini"> EP </span>
-                              <span class="sidebar-normal"> Edit Profile </span>
-                          </Link>
-                      </li>
-                      <li>
-                          <a href="#">
-                              <span class="sidebar-mini"> S </span>
-                              <span class="sidebar-normal"> Settings </span>
-                          </a>
-                      </li>
+                  <ul className="nav">
+                    <li>
+                      <Link to={`/${user.username}`}>
+                        <span className="sidebar-mini"> <i className="material-icons">person</i> </span>
+                        <span className="sidebar-normal"> My Profile </span>
+                      </Link>
+                    </li>
+                    <li>
+                      <Link to={`/${user.username}/edit`}>
+                        <span className="sidebar-mini"> <i className="material-icons">settings</i> </span>
+                        <span className="sidebar-normal"> Edit Profile </span>
+                      </Link>
+                    </li>
+                    <li>
+                      <a href="#" onClick={() => {
```

Comment:
I think the fact that this is both a link and has an onclick is not a good choice.  We need to stick with the onclick and remove the link.  Make a span or div.

Reply:
Yeah I know but when I changed it, it messed up the style :/